Basic FE project template/ui-src/styles/components/why-beleasing/_why-beleasing.scss

```diff
@@ -0,0 +1,159 @@
+.why-beleasing {
+  margin: 100px 100px;
+
+  @media screen and (max-width: 500px) {
+    margin: 55px;
+  }
+
+  &__container {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: start;
+
+    @media screen and (max-width: 1000px) {
+      display: flex;
+      flex-direction: column-reverse;
+      justify-content: center;
+      align-items: center;
+    }
+
+    h1 {
+      @media screen and (max-width: 530px) {
+        font-size: 36px;
+      }
+
+      @media screen and (max-width: 320px) {
+        font-size: 28px;
+      }
+    }
+  }
+
+  &__title {
+    max-width: 980px;
+    width: 100%;
+    margin-right: 10%;
+    margin-top: 30px;
+
+    @media screen and (max-width: 1000px) {
+      max-width: 500px;
+      margin: 25px 0;
+      text-align: center;
+    }
+
+    @media screen and (max-width: 530px) {
+      text-align: center;
+    }
+
+    @media screen and (max-width: 375px) {
+      text-align: center;
+    }
+  }
+
+  p {
```

Comment:
this looks like a global style because you targeted a "p" element. We have 2 options here: 1) we can move these styles int othe _globals.scss styles if you think this will be needed throughout the entire project, or 2) we can give this particular "p" a class in html and change this "p" here into that html class.